cli/tracker.php

```diff
@@ -17,6 +17,11 @@
 ini_set('display_errors', 1);
 
 define('JPATH_ROOT', realpath(__DIR__ . '/..'));
+define('JPATH_CONFIGURATION', JPATH_ROOT . '/etc');
```

Comment:
This seems to be missing in master so I believe we should merge the branch ASAP :wink: